users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -628,3 +629,138 @@ def run_mel_baseline():
         os.path.join(gs.ALIAS_AND_OUTPUT_SUBDIR, "report_mel_baseline.csv"),
         values=report.get_values(),
         template=report.get_template())
+
+def run_mel_audio_perturbation():
+    gs.ALIAS_AND_OUTPUT_SUBDIR = "experiments/switchboard/ctc/feat/"
+
+    (
+        returnn_datasets, rasr_loss_corpus_path, rasr_loss_corpus_segments, rasr_loss_lexicon_path, dev_corpora
+    ) = get_datasets(preprocessing=True)
+    returnn_args = {
+        "batch_size": 10000,
+        "rasr_binary_path": RASR_BINARY_PATH,
+        "rasr_loss_corpus_path": rasr_loss_corpus_path,
+        "rasr_loss_corpus_segments": rasr_loss_corpus_segments,
+        "rasr_loss_lexicon_path": rasr_loss_lexicon_path,
+        "datasets": returnn_datasets,
+    }
+    feature_args = {
+        "class": "LogMelNetwork",
+        "wave_norm": True,
+        "frame_size": 200,
+        "frame_shift": 80,
+        "fft_size": 256
+    }
+
+    nn_args, report_args_collection = get_nn_args_baseline(
+        nn_base_args={
+            "lgm80_conf-wei-oldspecaug-audio_perturbation": dict(
```

Comment:
Please stick to the args in `run_mel_baseline` except for the audio perturbation related parts. I think there are some differences.

Reply:
Done